bazel/genrule_repository.bzl

```diff
@@ -98,8 +105,8 @@ def _genrule_environment(ctx):
   lines.append("export CFLAGS=%r" % (" ".join(cc_flags),))
   lines.append("export LDFLAGS=%r" % (" ".join(ld_flags),))
   lines.append("export LIBS=%r" % (" ".join(ld_libs),))
-  lines.append("export CC=%r" % (ctx.var['CC'],))
-  lines.append("export CXX=%r" % (ctx.var['CC'],))
+  lines.append("export CC=%s" % (_absolute_bin(ctx.var['CC']),))
+  lines.append("export CXX=%s" % (_absolute_bin(ctx.var['CC']),))
```

Comment:
Why isn't this CXX?

Reply:
There is no `CXX` context variable, only `CC`. Bazel assumes that the same compiler command is used for both C and C++ code.